mackup/applications/yabai.cfg

```diff
@@ -3,3 +3,7 @@ name = yabai
 
 [configuration_files]
 .yabairc
+.config/yabai/yabairc
```

Comment:
As you added `xdg_configuration_files`, you can remove this, it would duplicate `.config/yabai/yabairc`.

Reply:
According to yabai's document, the path `$HOME/.config/yabai/yabairc` is also supported, and I use that as my config path. Although my `XDG_CONFIG_HOME` is empty.

Reply:
`.config/yabai/yabairc` is the default XDG config when `XDG_CONFIG_HOME` is unset.

Reply:
Thanks for the review. I just updated the file.